src/frontend/src/pages/SettingsPage/UserSettings/UserSettingsView.tsx

```diff
@@ -3,20 +3,24 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
-import { Grid, Link } from '@mui/material';
+import { Grid } from '@mui/material';
 import { UserSettings } from 'shared';
+import ExternalLink from '../../../components/ExternalLink';
 
 interface UserSettingsViewProps {
   settings: UserSettings;
 }
-
+const style = { display: 'flex', padding: 0 }
 const renderSlackId = (settings: UserSettings) => {
   return (
     <>
-      <b>Slack ID: </b>
-      <Link target="_blank" rel="noopener noreferrer" href={'https://nu-electric-racing.slack.com/team/' + settings.slackId}>
-        {settings.slackId}
-      </Link>
+      <div style={style}>
+        <b>Slack ID: </b>
+        <ExternalLink
+          link={'https://nu-electric-racing.slack.com/team/' + settings.slackId}
+          description={settings.slackId}
+        ></ExternalLink>
```

Comment:
For this External Link and the other one you use, because there aren't any child elements inside of them you can write them as `<ExternalLink link={} description={} />`, just a little cleaner

Also for the style prop you can do it this way or in cases like this where it's just one or two css prop you can do style ={{ display: 'flex', padding: 0 }} directly in the div. I'd reccomend doing it that way if there's only one css prop that's not likely to be changed later. It's good like this tho, just an fyi